src/main/java/uk/ac/ebi/subs/validator/data/structures/SingleValidationResultStatus.java

```diff
@@ -1,25 +1,19 @@
-package uk.ac.ebi.subs.validator.data;
+package uk.ac.ebi.subs.validator.data.structures;
 
 /**
- * Controlled vocabulary for validation status
+ * Controlled vocabulary for single validation result status
  *
  * <li>{@link #Pending}</li>
  * <li>{@link #Pass}</li>
  * <li>{@link #Error}</li>
  * <li>{@link #Warning}</li>
- * <li>{@link #Complete}</li>
  */
-public enum ValidationStatus {
+public enum SingleValidationResultStatus {
```

Comment:
You could remove the unneeded statuses from the JavaDoc

Reply:
There's no unneeded statuses here: Pending, Pass, Error and Warning.

Reply:
Sorry, I missed this one.